code/src/components/FourthQuestion.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react'
+import { OutputSummary } from './OutputSummary'
+import './fourthQuestion.css'
+
+export const FourthQuestion = ({ answer }) => {
+    const [activity, setActivity] = useState("")
+    const [hour] = useState("")
+    const [showSummary, setShowSummary] = useState("")
+
+    const handleSubmit = event => {
+        event.preventDefault()
+        setShowSummary(true)
+    }
+    return (
+        <section className="fourth-question" id="activity">
+            {!showSummary ? (
+                <form onSubmit={handleSubmit}>
+                    <h1>Choose one thing from you'll do for yourself today</h1>
+                    <h2>Keep hydrated and acknowledge your thoughts and feelings</h2>
+                    <select
+                        onChange={event => setActivity(event.target.value)}
+                        value={activity}
+                        required>
+                        <option value="" disabled>♡</option>
+                        <option value="try out a nice and relaxing meditatation">Meditate</option>
+                        <option value="drink more water, have a glass right now">Drink more water</option>
+                        <option value="go on a long walk, maybe go with a loved one">Go on a long walk</option>
+                        <option value="journal your thoughts and feelings on a regular basis">Journal your thoughts and feelings</option>
+                        <option value="enjoy a cup of tea">Enjoy a cup of tea</option>
+                        <option value="take that warm and cozy bath">Take a warm bath</option>
+                    </select>
+                    <button type="submit"
```

Comment:
One option might be to conditionally render/disable the buttons. So if the activity is not set, that's a false by default and you can use that either by:
`{activity && <button>}`
or
`<button disabled={!activity}`

Reply:
Same for the other components - forcing a state change before continuing